integration/rails_5.0_test.go

```diff
@@ -0,0 +1,155 @@
+package integration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testRails50(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		imageIDs     map[string]struct{}
+		containerIDs map[string]struct{}
+
+		name       string
+		source     string
+		buildpacks = []string{
+			settings.Buildpacks.MRI.Online,
+			settings.Buildpacks.Bundler.Online,
+			settings.Buildpacks.BundleInstall.Online,
+			settings.Buildpacks.NodeEngine.Online,
+			settings.Buildpacks.RailsAssets.Online,
+			settings.Buildpacks.Puma.Online,
+		}
+	)
+
+	it.Before(func() {
+		imageIDs = make(map[string]struct{})
+		containerIDs = make(map[string]struct{})
+
+		var err error
+		name, err = occam.RandomName()
+		Expect(err).NotTo(HaveOccurred())
+
+		source, err = occam.Source(filepath.Join("testdata", "5.0"))
+		Expect(err).NotTo(HaveOccurred())
+
+	})
+
+	it.After(func() {
+		for id := range containerIDs {
+			Expect(settings.Docker.Container.Remove.Execute(id)).To(Succeed())
+		}
+
+		for id := range imageIDs {
+			Expect(settings.Docker.Image.Remove.Execute(id)).To(Succeed())
+		}
+
+		Expect(settings.Docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+		Expect(os.RemoveAll(source)).To(Succeed())
+	})
+
+	it("creates a working OCI image with compiled rails assets", func() {
+		image, logs, err := settings.Pack.WithNoColor().Build.
+			WithBuildpacks(buildpacks...).
+			WithPullPolicy("never").
+			Execute(name, source)
+		Expect(err).NotTo(HaveOccurred(), logs.String())
+
+		imageIDs[image.ID] = struct{}{}
+
+		container, err := settings.Docker.Container.Run.
+			WithEnv(map[string]string{
+				"PORT":            "8080",
+				"SECRET_KEY_BASE": "some-secret",
```

Comment:
What is the purpose of `SECRET_KEY_BASE` in the integration tests?

Reply:
I see now after reading comment in the test data regarding secrets.

Reply:
Rails has the concept of a "secret key" that it uses to encrypt certain tokens. This value is required to be set to something in order for the app to start in production mode.